util/query.py

```diff
@@ -424,6 +441,7 @@ def _queryNftvolumes(
               symbol
               nft {
                 id
+                creator
```

Comment:
Great Q. 

The extra rewards are only going to creators (publishers), as per specified.

I added owners because I felt it would be useful for the frontend to show both owners and creators. Showing owners will be a complement to creators, and to help the user distinguish that one is not the other. It doesn't need to be in this PR, but it "fell out" conveniently from the rest of the work, hence it's here.

Reply:
wait, this breaks all ERC721 selling models.
Once the ownership of an erc721 has been transferred, all rights & rewards should be transferred to new owner  (all rights are being transferred now in erc721 template, so I don't see any reason to have rewards act differently)

Reply:
see my code suggestions below, because anyway, one line of code was broken